schema/sale/index.js

```diff
@@ -90,6 +90,12 @@ const SaleType = new GraphQLObjectType({
             .then(exclude(options.exclude, 'id'));
         },
       },
+      associated_sale: {
```

Comment:
it _could_ do with a `description` for other developers, 'associated' sale could be mysterious...  #minor